demos/common/stream_client/stream_client.py

```diff
@@ -199,26 +204,36 @@ def start(self, *, ovms_address : str, input_name : str, model_name : str, datat
             if self.height is None:
                 self.height = np_test_frame.shape[0]
         self.output_backend.init(self.sink, fps, self.width, self.height)
+
+        if streaming_api:
+            triton_client.start_stream(partial(self.callback, None, None, None))
             
         i = 0
         total_time_start = time.time()
-        while not self.force_exit:
-            timestamp = time.time()
-            frame = self.grab_frame()
-            if frame is not None:
-                np_frame = np.array([frame], dtype=datatype.dtype()) if batch else np.array(frame, dtype=datatype.dtype())
-                inputs=[grpcclient.InferInput(input_name, np_frame.shape, datatype.string())]
-                inputs[0].set_data_from_numpy(np_frame)
-                triton_client.async_infer(
-                    model_name=model_name,
-                    callback=partial(self.callback, frame, i, timestamp),
-                    inputs=inputs)
-                i += 1
-            if limit_stream_duration > 0 and time.time() - total_time_start > limit_stream_duration:
-                break
-            if limit_frames > 0 and i > limit_frames:
-                break
-        self.pq.put((i, "EOS"))
+        try:
+            while not self.force_exit:
+                timestamp = time.time()
+                frame = self.grab_frame()
+                if frame is not None:
+                    np_frame = np.array([frame], dtype=datatype.dtype()) if batch else np.array(frame, dtype=datatype.dtype())
+                    inputs=[grpcclient.InferInput(input_name, np_frame.shape, datatype.string())]
+                    inputs[0].set_data_from_numpy(np_frame)
+                    if streaming_api:
+                        triton_client.async_stream_infer(model_name=model_name, inputs=inputs, parameters={"OVMS_MP_TIMESTAMP":int(cv2.getTickCount() / cv2.getTickFrequency() * 1e6)})
```

Comment:
```suggestion
                        triton_client.async_stream_infer(model_name=model_name, inputs=inputs, parameters={"OVMS_MP_TIMESTAMP":int(cv2.getTickCount() / cv2.getTickFrequency() * 1e6)})
```

extract method calculate timestamp:
int(cv2.getTickCount() / cv2.getTickFrequency() * 1e6)